admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -765,7 +771,7 @@ def _makeIndex(self, database, table):
         self._log.info('Generating index %r.%r', self.indexDb, metaTable)
 
         # try to delete existing table first
-        self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
+        #self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
```

Comment:
I'm OK with dropping DROP TABLE if it works both for you and integration tests. What I'm saying is that it works by pure accident now and we should try not to break this accidental feature (or try to make it more official).